code/modules/projectiles/projectile/special/wormhole.dm

```diff
@@ -20,7 +20,8 @@
 	if(casing)
 		gun = casing.gun
 
+
 /obj/item/projectile/beam/wormhole/on_hit(atom/target)
 	if(!gun)
-		qdel(src)
+		return qdel(src)

```

Comment:
qdel does not have any meaningful return value, so this should be on next line to not confuse people reading this later.